src/Switch/Switch.tsx

```diff
@@ -5,27 +5,26 @@ type Props = {
   disabled?: boolean;
   id: string;
   currentValue: boolean;
-  small: boolean;
+  size?: 'small' | 'default';
```

Comment:
If you pass in default, is it the same as not passing in anything at all?

Reply:
yeah.

Reply:
Do we want to make it so it defaults to 'defsult' and they override it with small rather than can pass in default that doesn't do anything?

Reply:
I think it is a pretty common practice to have all the options named out, and then defaulting to one of those. Maybe the name default should change? to normal?